docs/cave/goes-satellite.md

```diff
@@ -0,0 +1,308 @@
+
+![](../images/goes-wmeso12.png)
+
+### Alaska
+
+![](../images/goes-ak.png)
+
+### Hawaii
+
+![](../images/goes-hi.png)
+
+### Puerto Rico (PRREGI)
+
+![](../images/goes-pr.png)
+
+---
+
+## RGB Composites
+
+RGB Composites are made by combining 3 channels and are available for each sector. Quite a few new RGB products have been added in Unidata's 18.2.1 release. These products are generated **on the fly in AWIPS** using the existing channel products from EDEX.
+
+!!! warning "GOES RGB Imagery is NOT SUPPORTED on macOS or within a Virtual Machine"
+
+	OpenGL Shading Language limitations prevent multi-channel imagery from displaying correctly on Mac or in a Virtual Machine.  Please use the Linux or Windows installs to view RGB products.
+
+### Day Cloud Phase
+
+![](../images/RGB-01-DayCloudPhase.png)
+
+### Fire Temperature
+
+![](../images/RGB-02-FireTemperature.png)
+
+### Day Land Cloud
+
+![](../images/RGB-03-DayLandCloud.png)
+
+### Day Cloud Convection
+
+![](../images/RGB-04-DayCloudConvection.png)
+
+### Day Land Cloud Fires
+
+![](../images/RGB-05-DayLandCloudFires.png)
+
+### VIS/IR Sandwich
+
+![](../images/RGB-06-VISIRSandwich.png)
+
+### Simple Water Vapor
+
+![](../images/RGB-07-SimpleWaterVapor.png)
+
+### Air Mass
+
+![](../images/RGB-08-AirMass.png)
+
+### Ash 
+
+![](../images/RGB-09-Ash.png)
+
+### Day Convection
+
+![](../images/RGB-10-DayConvection.png)
+
+### Day Snow Fog
+
+![](../images/RGB-11-DaySnowFog.png)
+
+### Differential Water Vapor
+
+![](../images/RGB-12-DifferentialWaterVapor.png)
+
+### Dust
+
+![](../images/RGB-13-Dust.png)
+
+### CIMSS Natural Color
+
+![](../images/RGB-14-CIMSSNaturalColor.png)
+
+### Nighttime Microphysics
+
+![](../images/RGB-15-NighttimeMicrophysics.png)
+
+### SO2
+
+![](../images/RGB-16-SO2.png)
+
+### CIRA Geocolor
+
+![](../images/RGB-17-CIRAGeocolor.png)
+
+### CIRA Debra Dust
+
+![](../images/RGB-18-CIRADebraDust.png)
+
+### CIRA Cloud Snow
+
+![](../images/RGB-19-CIRACloudSnow.png)
+
+### Daytime Composite 1
+
+![](../images/RGB-20-DaytimeComp1.png)
+
+### Daytime Composite 5
+
+![](../images/RGB-21-DaytimeComp5.png)
+
+---
+
+## Channel Differences
+
+Channel differences are the result of subtracting one channel from another to produce a new product.  These products are generated **on the fly in AWIPS** using the existing channel products from EDEX.
+
+There currently 10 channel differences that are offered in CAVE:
+
+* Split Window (10.3 - 12.3 &mu;m)
+* Split Cloud Top Phase (11.2 - 8.4 &mu;m)
+* Night Fog (10.3 - 2.9 &mu;m)
+* Day Fog (3.9 - 10.3 &mu;m)
+* Split Fire (2.2 - 1.6 &mu;m)
+* Split Ozone (9.6 - 10.3 &mu;m)
+* Split Water Vapor (6.19 - 7.3 &mu;m)
+* Split Snow (1.6 - 0.64 &mu;m)
+* Vegetation (0.64 - 0.87 &mu;m)
+* Upper Level Info (11.2 - 6.19 &mu;m)
+
+!!! note "The rendering of these products uses the **Jep** package in Python, which has known to have issues installing on the MacOS and Windows versions."
```

Comment:
I'm not sure I'd word it quite this way... Maybe say which need to be installed using very specific instructions found on our [installation page](link to our installation page).

Reply:
Sure, I only updated the "formatting" of the note, the wording was what existed previously. Do we have problems with jep in Mac now - I didn't think so? New wording:

"The rendering of these products uses the **Jep**  package in Python, which has specific installation steps (link to instructions) for Windows"